docs/content/latest/yugabyte-platform/create-deployments/create-universe-multi-zone.md

```diff
@@ -110,7 +106,13 @@ You connect to a database node as follows:
 
 Yugabyte Platform includes a number of sample applications enclosed in Docker containers. 
 
-To access instruction on how to run sample applications, select your universe's **Overview** and then click **Actions > Run Sample Apps** to open the Run Sample Apps dialog shown in the following illustrations:
+The jdbc connection string is only needed if you're hooking up a Java application to the database. 
+
+The YSQL, YCQL, and Yedis endpoints are passed into the driver to let it know what IP addresses and ports to use to connect to the database. 
+
+The sample apps include the endpoint information in the command that is run on the platform node anyway.
+
```

Comment:
I recommend removing these lines. My comments were indented as feedback - this information isn't relevant to this section of the docs..

Reply:
@ionthegeek  Oh, sorry, these weren't meant to be in the doc. It was a copy-paste error. Thanks for catching it.